### DISCUSSION TITLE
Access interactive shell in docker container

### DISCUSSION THREAD

**Discussion by 0xAl3xH** [bluenviron/mediamtx#2190] [Questions]
Is there a way to access a shell inside `bluenviron/mediamtx:latest` I have tried to run 

`docker exec -it {container id here} {sh | bash | /bin/bash | /bin/sh}` to no avail. 

Secondly, is the dockerfile used to build this container available somewhere?

**Comment by kewne7768** [comment]
The regular `latest` tag is just the app itself and absolutely nothing else, the `latest-ffmpeg` tag is based on Alpine and includes a shell, package manager, a functional SSL root cert store (needed if `externalAuthenticationURL` is https) and all the usual stuff you'd expect. Whatever you're trying to do is probably best done by just switching to the ffmpeg image for use or as base for your own container image.

The build stuff is a little bit unusual. Look in the repo's main `Makefile`, `scripts/binaries.mk` and `scripts/dockerhub.mk`; the `binaries.mk` does the build and `dockerhub.mk` seemingly makes the images. I nearly assembled a custom build myself before noticing that the -ffmpeg one did the job I needed it to.

[a comment by an automated account is not shown]
